### ISSUE TITLE
Is issue #833 back again? 

### ISSUE THREAD

**Issue by richb201** [FusionAuth/fusionauth-php-client#22]
I just added a third tenant and i am seeing this:
A 404 status code is returned from the Start Passwordless API when more than one tenant exists in FusionAuth.

This was solved in "Resolves GitHub Issue #833, thanks to @atrauzzi for reporting and helping us track this one down!"
This was reported solve din 1.19.0. I am using 1.19.7 and also experience this (it is repeatable). Can this be?

Please advise.

**Comment by robotdan** [comment]
Can you confirm you are using the 1.19.7 version of the client and the FusionAuth server? 
Please provide an example JSON request that is causing a `404`.

**Comment by richb201** [comment]
Hi Dan. Here is a screenshot of the UI showing vers 1.19.7.
Here is the call and the response. You can see the request in the json of
the call:

Here is the call:
$result =
$_SESSION['client']->withTenantId('8ea1c784-866b-4755-b97b-b4fda2ad19e4')->startPasswordlesslogin($requestJ);

Here is the $result I am seeing:
FusionAuth\ClientResponse Object
(
[errorResponse] =>
[exception] =>
[method] => POST
[request] => {"
applicationId":"2cf00c29-ac46-49bf-8cd4-32538ddb00d8","loginId":"richardbernstein217@gmail.com
","state":{"redirect_uri":"
http://substantiator-survey.ngrok.io/index.php/Configure/report_generator_amazing","client_id":"2cf00c29-ac46-49bf-8cd4-32538ddb00d8","response_type":"code","scope":"openid","state":"richardbernstein217@gmail.com
"}}
[successResponse] =>
[status] => 404
)


We were trying to nail down the problem to the fact that I am using
multiple tenants, but the 404 response is returned regardless.

**Comment by robotdan** [comment]
The only reason `404` would be returned is if the `applicationId` of `2cf00c29-ac46-49bf-8cd4-32538ddb00d8` does not exist or is disabled, or if a user `richardbernstein217@gmail.com`.

You can confirm the user exists, and the `applicationId` is correct, and ensure the `loginId` does not contain any trailing whitespace or other characters. 

Does the issue go away if you provide a `tenantId` on the request?

**Comment by richb201** [comment]
No, the problem did not go away with a tenantId. I don't think the appid is disabled (i will check). I will also check out to make sure the user exists and there is no trailing spaces, tomorrow.

You saw the version in that image of the UI? Is that a confirmation of the actual a version?

On December 29, 2020, at 18:41, Daniel DeGroff <notifications@github.com> wrote:

The only reason 404 would be returned is if the applicationId of 2cf00c29-ac46-49bf-8cd4-32538ddb00d8 does not exist or is disabled, or if a user richardbernstein217@gmail.com.

You can confirm the user exists, and the applicationId is correct, and ensure the loginId does not contain any trailing whitespace or other characters.

Does the issue go away if you provide a tenantId on the request?

—
You are receiving this because you authored the thread.
Reply to this email directly, view it on GitHub, or unsubscribe.￼

[ { "@context": "<a href="http://schema.org">http://schema.org</a>", "@type": "EmailMessage", "potentialAction": { "@type": "ViewAction", "target": "<a href="https://github.com/FusionAuth/fusionauth-php-client/issues/22#issuecomment-752277378">https://github.com/FusionAuth/fusionauth-php-client/issues/22#issuecomment-752277378</a>", "url": "<a href="https://github.com/FusionAuth/fusionauth-php-client/issues/22#issuecomment-752277378">https://github.com/FusionAuth/fusionauth-php-client/issues/22#issuecomment-752277378</a>", "name": "View Issue" }, "description": "View this Issue on GitHub", "publisher": { "@type": "Organization", "name": "GitHub", "url": "<a href="https://github.com">https://github.com</a>" } } ]

**Comment by robotdan** [comment]
I don't think your screenshot came through, you may want to comment directly on the issue, I think the email responses lose some things. https://github.com/FusionAuth/fusionauth-php-client/issues/22

**Comment by richb201** [comment]
Ok, thanks. Yes. On the Home/System/About screen the version shows as 1.19.7

**Comment by richb201** [comment]
Dan, I checked the loginId which is "richardbernstein217@gmail.com".
You can see that there are no extra spaces in that. I have copied the
"source" from the UI below. The email address is a valid loginId, right? I
also checked the application ID and it is fine. Please advise.

{
  "active" : true,
  "connectorId" : "e3306678-a53a-4964-9040-1c96f36dda72",
  "data" : {
    "admin_email" : "richardbernstein217@gmail.com",
    "email" : "richardbernstein217@gmail.com"
  },
  "email" : "richardbernstein217@gmail.com",
  "id" : "15418631-8d91-4acf-afee-1e5312f2fb2b",
  "insertInstant" : 1609037772287,
  "lastUpdateInstant" : 1609272795838,
  "passwordChangeRequired" : false,
  "passwordLastUpdateInstant" : 1609037772351,
  "registrations" : [ {
    "applicationId" : "95b1cc05-cad3-496e-9fc5-e8e3521b1b85",
    "id" : "a9fd1cce-7831-4e0b-b3ff-80dd7521c289",
    "insertInstant" : 1609037772361,
    "lastLoginInstant" : 1609037772363,
    "lastUpdateInstant" : 1609037772361,
    "usernameStatus" : "ACTIVE",
    "verified" : true
  } ],
  "tenantId" : "34303566-6466-3234-6565-656233663263",
  "twoFactorDelivery" : "None",
  "twoFactorEnabled" : false,
  "usernameStatus" : "ACTIVE",
  "verified" : true
}
